### DISCUSSION TITLE
Allow to connect to Kong API Loopback

### DISCUSSION THREAD

**Discussion by nasrulhazim** [Kong/kong-manager#154] [Ideas]
Hi, 

I would love to have the Kong Manager to be connect to the Kong API Loopback as documented in this section [Kong API Loopback](https://docs.konghq.com/gateway/latest/production/running-kong/secure-admin-api/#kong-api-loopback).

This allows the Kong Admin API to be exposed securely to the Kong Manager.

At the moment I've forked the repo and try to allow API key to be pass in the configuration - but a bit stuck how to get the info from .env file.

Looking forward for this feature to be implemented - and at the same time try to work on customisation  in the fork repository.

**Comment by pdany1116** [comment]
Hi @nasrulhazim, did you manage to make this work?

I configured the admin api with [Kong API Loopback](https://docs.konghq.com/gateway/latest/production/running-kong/secure-admin-api/#kong-api-loopback) and secured it with a query param api key with the [key-auth plugin](https://developer.konghq.com/plugins/key-auth/). I tried tricking Kong Manager by setting the `admin_gui_api_url=http://127.0.0.1/admin-api?api_key=abc`, but it appends the routes to the url, resulting in invalid URLs `http://127.0.0.1/admin-api?api_key=abc/routes`.
